include/deal.II/base/numbers.h

```diff
@@ -423,12 +443,20 @@ namespace numbers
 
   // --------------- inline and template functions ---------------- //
 
+  template <typename Number>
+  struct is_cuda_compatible<std::complex<Number>, void> : std::false_type
+  {};
```

Comment:
I would move this right below the general declaration

Reply:
In particular, it ought to be above the uses in `abs_square`.

Reply:
Done